Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -155,15 +159,16 @@ def bump_formula_pr
       new_url.sub "mirrors.ocf.berkeley.edu/debian", "mirrorservice.org/sites/ftp.debian.org/debian"
     end
     new_mirrors ||= [new_mirror] unless new_mirror.nil?
-    new_version = args.version
     old_url = formula_spec.url
     old_tag = formula_spec.specs[:tag]
     old_formula_version = formula_version(formula, requested_spec)
     old_version = old_formula_version.to_s
-    forced_version = false
+    forced_version = !new_version.nil?
```

Comment:
```suggestion
    forced_version = new_version.present?
```
We have activesupport so this should work and is clearer than a double negation.

Reply:
Oh that's much better, see anything else that needs changing?